acsoo/templates/project/+project.name+/.gitlab-ci.yml.bob

```diff
@@ -81,16 +81,12 @@ test:
     - virtualenv --python=$PYTHON venv
     - venv/bin/pip install coverage
     # use --no-index so missing dependencies that would not be in *.whl are detected
-    - venv/bin/pip install --no-index release/*.whl
-    # although the project is part of *.whl, install it in editable mode so coverage sees it
-    - venv/bin/pip install --no-index -e .
+    # install the project in editable mode (-e) so coverage sees it
+    - venv/bin/pip install --no-index --find-links release -e odoo-addons-{{{ project.name }}}
```

Comment:
```suggestion
    - venv/bin/pip install --no-index --find-links release -e . odoo-addons-{{{ project.name }}}
```

Reply:
Indeed. And the project name is therefore not needed.